Defs/Ammo/Advanced/PlasmaGasCartridge.xml

```diff
@@ -0,0 +1,469 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Defs>
+
+    <ThingCategoryDef>

```

Comment:
The `Defs/Ammo/` library is generally only for real-world calibers, or genericized sci-fi ammo shared across several different mods.

For non-generic faction-specific sci-fi ammo (such as Star Wars factions), I strongly suggest moving them into the `Patches/Star Wars - Factions` folder and re-implement them as PatchOperations instead, so that CE's ammo patcher on game startup isn't unnecessarily checking for Star Wars ammo if the Star Wars Factions isn't active.

Each ammo family should ideally also be a separate XML file, to make it easier to maintain the patches.

See the DOOM patch folder for examples of how to do this.

Reply:
Resolved by moving ammo defs to `Patches/Star Wars - Factions/PlasmaGasCartridge.xml`